js/gui.js

```diff
@@ -19,6 +19,8 @@ window.addEventListener("load", function()
     resolve({
       "characters/robo.svg": "Robo",
       "characters/swammy.svg": "Swammy",
+	  "characters/monster.svg": "Monster",
```

Comment:
intend it with one more space

Reply:
:100: Everywhere happening :sweat_smile:

Reply:
@0x48piraj make sure you don't use `tabs`, that might have been the cause. Spaces are more better, configure your editor to use soft tabs instead of tabs if u really like them.

Reply:
@abishekvashok 
Thanks !! :fire: 
I love who make me GSearch 
> Reading : https://stackoverflow.com/questions/26350689/what-are-hard-and-soft-tabs

Reply:
:)